sktime/forecasting/adapters/_hcrystalball.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Adapter for using HCrystalBall forecastsers in sktime."""
+
+__author__ = ["MichalChromcak"]
+
+import pandas as pd
+from sklearn.base import clone
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("hcrystalball", severity="warning")
```

Comment:
This is not in `all_extras`

Reply:
if you read the PR description, it says:

> `hcrystalball` is not added as soft dependency in `all_extras`, which is fine, as `sktime` now also has full capability for dependency isolation.

(fine in my opinion)